main.tf

```diff
@@ -33,29 +33,49 @@ module "resource_group" {
 
   name     = local.resource_group_name
   location = var.location
-  tags     = local.resource_group_tags
+  tags     = local.tags
 }
 
 module "network" {
-  source  = "d2lqlh14iel5k2.cloudfront.net/module_primitive/virtual_network/azurerm"
-  version = "~> 2.0"
-
-  resource_group_name                              = module.resource_group.name
-  vnet_location                                    = var.location
-  vnet_name                                        = local.virtual_network_name
-  address_space                                    = var.address_space
-  subnet_names                                     = var.subnet_names
-  subnet_prefixes                                  = var.subnet_prefixes
-  bgp_community                                    = var.bgp_community
-  ddos_protection_plan                             = var.ddos_protection_plan
-  dns_servers                                      = var.dns_servers
-  nsg_ids                                          = var.nsg_ids
-  route_tables_ids                                 = var.route_tables_ids
-  subnet_delegation                                = var.subnet_delegation
-  subnet_service_endpoints                         = var.subnet_service_endpoints
-  subnet_private_endpoint_network_policies_enabled = var.subnet_private_endpoint_network_policies_enabled
-  tags                                             = local.vnet_tags
-  use_for_each                                     = var.use_for_each
+  source = "git::https://github.com/launchbynttdata/tf-azurerm-module_primitive-virtual_network.git?ref=fix/route-table-association"
```

Comment:
can you create a PR for this change to the primitive?

Reply:
https://github.com/launchbynttdata/tf-azurerm-module_primitive-virtual_network/pull/6